juntagrico/dao/jobdao.py

```diff
@@ -76,5 +76,10 @@ def get_promoted_jobs():
                                           time__gte=timezone.now()).order_by('time')[:Config.promomted_jobs_amount()]
 
     @staticmethod
-    def upcomming_jobs_for_member(member):
-        return Job.objects.filter(time__gte=timezone.now(), assignment__member=member).distinct()
+    def upcoming_jobs_ordered_by_time_ascending_for_member(member):
+        upcoming_jobs = Job.objects\
```

Comment:
Is there a benefit for having a temporary variable? I would keep it concise with a return here.

Reply:
OK, I'll do that.